src/main/java/org/jboss/ejb/client/ConfigurationBasedEJBClientContextSelector.java

```diff
@@ -41,13 +45,17 @@
  * A one-time, configuration-based EJB client context configurator.
  *
  * @author <a href="mailto:david.lloyd@redhat.com">David M. Lloyd</a>
+ * @author <a href="mailto:jbaesner@redhat.com">Joerg Baesner</a>
  */
 final class ConfigurationBasedEJBClientContextSelector {
     private static final EJBClientContext configuredContext;
 
     private static final String NS_EJB_CLIENT_3_0 = "urn:jboss:wildfly-client-ejb:3.0";
+    private static final String NS_EJB_CLIENT_3_1 = "urn:jboss:wildfly-client-ejb:3.1";
     private static final String NS_INCORRECT = "urn:jboss:ejb-client:3.0";
 
+    private static final Set<String> validNamespaces = Stream.of(NS_EJB_CLIENT_3_0, NS_EJB_CLIENT_3_1).collect(Collectors.toCollection(HashSet::new));
```

Comment:
Stylistically, I'd prefer to avoid the lambda and just do `new HashSet<>(List.of(NS_EJB_CLIENT_3_0, NS_EJB_CLIENT_3_1))`.

Reply:
I believe you mean `new HashSet<>(Arrays.asList(NS_EJB_CLIENT_3_0, NS_EJB_CLIENT_3_1))`

Reply:
Yes, right.